packages/@uppy/core/src/Uppy.js

```diff
@@ -972,6 +972,19 @@ class Uppy {
       }
     })
 
+    let uploadStalledWarningRecentlyEmitted
+    this.on('upload-stalled', (error, files) => {
+      const { message } = error
+      const details = files.map(file => file.meta.name).join(', ')
```

Comment:
Think is going to look bad as soon as you have more than 10 files. I would say this is a general problem in `informer`. It's probably best if we could inline errors per file always and use infomer only for generic updates. Any chance we can reuse the meta data validation?

Reply:
Note that the details is not shown by default, it's only visible if the user click on the `?` on the informer bubble.